examples/pubsub/discord/main.go

```diff
@@ -17,7 +17,7 @@ type discordMessage struct {
 	AvatarURL string `json:"avatar_url"`
 	TTS       bool   `json:"tts"`
 	File      []byte `json:"file"`
-	Embeded   []byte `json:"embeds"`
+	Embedded   []byte `json:"embeds"`
```

Comment:
This struct changed the field name. Doesn't seem to impact the code.
But worth to check in detail

Reply:
is not used currently, only text is sent right now